packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -489,6 +489,20 @@ export class StripeHelper {
       },
       { idempotencyKey: `ssc-${subIdempotencyKey}` }
     );
+
+    const paymentIntent = (subscription.latest_invoice as Stripe.Invoice)
```

Comment:
@chenba what do you think. Something like this? (I'll clean it up still)

Reply:
I don't actually know whether Stripe has already attempted a charge at this point.  If it did, then this probably should work.

Reply:
From the few examples that I've seen and from what the discord chat, it does look like a charge has been attempted.